src/badguy/walking_badguy.hpp

```diff
@@ -18,6 +18,7 @@
 #define HEADER_SUPERTUX_BADGUY_WALKING_BADGUY_HPP
 
 #include "badguy/badguy.hpp"
+#include "supertux/tile.hpp"
```

Comment:
Do you need this include? Can it go into badguy.cpp?

Reply:
when it wasn't the compiler complained

Reply:
```/home/noe/Bureau/SuperTux/supertux/src/badguy/walking_badguy.cpp: In member function ‘virtual void WalkingBadguy::collision_tile(uint32_t)’:
/home/noe/Bureau/SuperTux/supertux/src/badguy/walking_badguy.cpp:196:26: error: ‘Tile’ has not been declared
   if( (tile_attributes & Tile::HURTS)  &&  max_drop_height > -1  &&  on_ground())